creator-node/src/resizeImage.js

```diff
@@ -96,4 +103,84 @@ function _exifRotate (img, exif) {
   return img
 }
 
-module.exports = resizeImage
+module.exports = async (job) => {
+  const {
+    file,
+    fileName,
+    storagePath,
+    sizes,
+    square,
+    logContext
+  } = job.data
+  const ipfs = ipfsClient(
```

Comment:
we probably can't pass this ipfs connection through right? probably won't serialize right? don't know if this would leak ipfs connections, but i'm assuming there's some cleanup mechanism so we don't need to worry about this

Reply:
yeah i think we're good here, it should be deallocated